infrastructure/output.tf

```diff
@@ -1,9 +1,9 @@
 output "vaultName" {
-  value = module.civil_sdt_key_vault.key_vault_name
+  value = data.azurerm_key_vault.key_vault.name
```

Comment:
Change key_vault_id value to module.civil_sdt_key_vault.key_vault_id as secret is in the civil-sdt key vault.

Reply:
changed